# Robot parameters
tcp_speed: 0.03

# Gripper parameters
tcp2fingertip: 0.32455 # distance from tcp to gripper fingertip
opening_per_count: 0.00065 # gripper stroke opening per rPr count
finger_thickness: 0.008
max_opening: 0.3 # max stroke of gripper excluding finger thickness
gripper_speed: 0.1 # value between 0.013 and 0.100
gripper_force: 10 # value between 0 and 100

# Object dimension
object_thickness: 0.007  # object thickness in meters 
object_length: 0.022 # object length in meters

# Initial configuration
delta_0: 0.014  # distance from fingertip to object tip within gripper 0.0185
theta_0: 47

# Intermediate configuration
psi_regrasp: 58 
theta_tilt: 15 
tuck: 5 

# Action axis
axis: [1, 0, 0]

# Simulation
sim:
